tools/pipelines/templates/include-test-perf-benchmarks.yml

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# This template is specifically intended to be used for common steps required across different stages
+# of test-perf-benchmarks.yml pipeline, and assumes it has access to variables that have been defined/imported
```

Comment:
Nit: would it make more sense to have the necessary variables passed in as parameters to make the contract a bit more explicit?

Reply:
Hmm yes... there was a reason why I didn't do it but I can't recall what it was so it must not have been very good 😅 . I'll update.